src/tribler/core/utilities/process_checker.py

```diff
@@ -28,9 +31,9 @@ class ProcessChecker:
     This class contains code to check whether a Tribler process is already running.
     """
 
-    def __init__(self, directory: Path, lock_file_name: str = 'tribler.lock'):
+    def __init__(self, directory: Path, lock_file_name: Optional[str] = None):
         self.logger = logging.getLogger(self.__class__.__name__)
-        self.lock_file = directory / lock_file_name
+        self.lock_file = directory / (lock_file_name or LOCK_FILE_NAME)
```

Comment:
NIT: from my POV this version is easier to read and modify:

```suggestion
        lock_file_name = lock_file_name or LOCK_FILE_NAME 
        self.lock_file = directory / lock_file_name
```

Reply:
I agree